casper/src/main/scala/coop/rchain/casper/BlockRandomSeed.scala

```diff
@@ -1,3 +1,7 @@
+package coop.rchain.casper
```

Comment:
The changes in this file should not be the final result. Currently the changes here is to make the code compile and tests pass. The final result should be taken in https://github.com/rchain/rchain/pull/3729 . This pr would be merged after #3729 .